media-source/mediasource-remove.html

```diff
@@ -13,7 +13,9 @@
           {
               var sourceBuffer = mediaSource.addSourceBuffer(MediaSourceUtil.AUDIO_VIDEO_TYPE);
 
-              assert_throws("InvalidAccessError", function()
+              mediaSource.duration = 10;
```

Comment:
aside: Why do we need to set positive duration here? It shouldn't matter to this test. Since it doesn't and we've been slow on review, I'll incorporate this part of this into the merge result. Note that InvalidAccessError->TypeError check here is already in the suite.